### PR TITLE
Night mode and map colors

### PR DESCRIPTION
**Pull request by Vrabbers**

### CODE REVIEW

**Comment by Vrabbers** [comment]
The change in the way the lines are rendered would allow the gradients to transform but for some reason it still doesn't work

**Review by Royce551 — APPROVED** [review]
LGTM

File: BnbnavNetClient/Views/MapView.axaml.cs [hunk 1]
```diff
@@ -90,37 +91,70 @@ protected override void OnInitialized()
...
+                RoadType.Roundabout => this.FindResource("RoundaboutRoadPen")!,
+                RoadType.DuongWarp => this.FindResource("DuongWarpRoadPen")!,
+                _ => this.FindResource("UnknownRoadPen")!,
+            });  
             var fromEdge = mapService.Nodes[edge.From.Id];
             var from = ToScreen(new(fromEdge.X, fromEdge.Z));
             var toEdge = mapService.Nodes[edge.To.Id];
             var to = ToScreen(new (toEdge.X, toEdge.Z));
+
+            static double Angle(Point a, Point b)
```
Comment (vicr123) [inline BnbnavNetClient/Views/MapView.axaml.cs]: Are these functions required? We only use them once so maybe we can inline them, or maybe they're useful elsewhere so maybe we can take it out into a helper class or something?

**Review by vicr123 — COMMENTED** [review]
With Comments

**Pull request merged**